pkg/api/api.go

```diff
@@ -102,6 +102,7 @@ type IssueCodeRequest struct {
 	SymptomDate string `json:"symptomDate"` // ISO 8601 formatted date, YYYY-MM-DD
 	TestDate    string `json:"testDate"`
 	TestType    string `json:"testType"`
+	TZOffset    int    `json:"tzOffset"` // offset in minutes of the user's timezone.
```

Comment:
Can we document what happens if this is omitted? i.e. you're speaking UTC.
And also I think direction matters here for documentation. What sign is required.

like we're on the negative side on UTC, but if I add a negative TZ offset, The logic in issue.go will put us further away

Reply:
I believe we're all set. I've added unit tests to help validate. LMK.